polyfills/document.querySelector/polyfill.js

```diff
@@ -4,7 +4,7 @@
 
 	function getElementsByQuery(node, selector, one) {
 		var
-		generator = document.createElement(),
+		generator = document.createElement('null'),
```

Comment:
Is this valid?  Is it worth using a real element to avoid possible rarely tested edge cases?